lib/pengine/bundle.c

```diff
@@ -1078,10 +1084,10 @@ pe__unpack_bundle(pe_resource_t *rsc, pe_working_set_t *data_set)
         clear_bit(rsc->flags, pe_rsc_unique);
     }
 
-    bundle_data->docker_run_command = crm_element_value_copy(xml_obj, "run-command");
-    bundle_data->docker_run_options = crm_element_value_copy(xml_obj, "options");
+    bundle_data->container_command = crm_element_value_copy(xml_obj, "run-command");
+    bundle_data->runtime_options = crm_element_value_copy(xml_obj, "options");
```

Comment:
why not container_options here for consistency and question if runtime should be used to specify docker/podman/rocket

Reply:
container_command is the command to run inside the container, so I think container_options would actually be more confusing since it's not the options to that command.

If we stick with runtime then runtime_options is good, otherwise maybe launcher_options or even just options since that's the configuration attribute name

Reply:
true ... then whatever we decide to use instead of runtime then is good in here as well ...